data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -318,6 +318,44 @@ then
     record_service_stage_success
 fi
 
+if [ ! -f cmdbootline-nto-bootstrap.done ]; then
+	record_service_stage_start "cmdbootline-nto-bootstrap"
+
+	rm --recursive --force cmdbootline-nto-bootstrap
+
+	# Create folders for overlay and podman command to use them
+	tmpfolder=$(mktemp -d)
+	echo ${tmpfolder}
+	podmancmd="bootkube_podman_run -d --rm "
```

Comment:
```suggestion
	podmancmd="bootkube_podman_run --attach=stderr --rm "
```
This was tested and does not exhibit the issue of https://issues.redhat.com/browse/CNF-10610

Reply:
Thanks for the comment.
Changed.